backend/danswer/document_index/vespa_constants.py

```diff
@@ -0,0 +1,85 @@
+from danswer.configs.app_configs import VESPA_CONFIG_SERVER_HOST
```

Comment:
Moved constants that were only used with vespa into this file for better organization